pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -645,7 +645,7 @@ func (m *migrationMonitor) isMigrationProgressing(domainSpec *api.DomainSpec) bo
 	// check if the migration is progressing
 	progressDelay := now - m.lastProgressUpdate
 	if m.progressTimeout != 0 && progressDelay/int64(time.Second) > m.progressTimeout {
-		logger.Warningf("Live migration stuck for %d sec", progressDelay)
+		logger.Warningf("Live migration stuck for %d sec", m.progressTimeout)
```

Comment:
isn't `progressDelay` the accurate length of time migration logic waited for progress? `m.progressTimeout` is the threshold, but `progressDelay` is the reality of how long was actually waited.  

I think the issue is that you want to see the progress delay in seconds. so maybe `progressDelay/int64(time.Second)` gives you that?

Reply:
ahh right... we just have to log  progressDelay in seconds.